web/src/invite.js

```diff
@@ -190,17 +190,12 @@ function valid_to(expires_in) {
         return $t({defaultMessage: "Never expires"});
     }
     const valid_to = add(new Date(), {minutes: time_valid});
-    const options = {
-        year: "numeric",
-        month: "numeric",
-        day: "numeric",
-        hour: "2-digit",
-        minute: "2-digit",
-        hour12: !user_settings.twenty_four_hour_time,
-    };
+    const date = timerender.get_localized_date_or_time_for_format(valid_to, "dayofyear_year");
+    const time = timerender.get_localized_date_or_time_for_format(valid_to, "time");
+
     return $t(
         {defaultMessage: "Expires on {date}"},
```

Comment:
I think for translators it'll be easier to handle and conjugate correctly if we make this string `Expires at {date} at {time}` with the two values passed in here. (It may compose OK even if we don't, but it's more obviously easy for translators to manage this way)

Reply:
Did you mean `Expires on {date} at {time}`?

Reply:
Yes, I think Tim meant `Expires on {date} at {time}`. I have made the necessary changes to have that string, so it should work properly in English now. We'll need to translate this string into other languages after this commit.